src/test/java/net/minecraftforge/debug/fluid/NewFluidTest.java

```diff
@@ -103,6 +103,16 @@ public void registerFluids(RegistryEvent.Register<Fluid> event)
         event.getRegistry().registerAll(test_fluid, test_fluid_flowing);
     }
 
+    public void loadComplete(FMLLoadCompleteEvent event)
+    {
+        // some sanity checks
+        BlockState state = Fluids.WATER.getDefaultState().getBlockState();
+        BlockState state2 = Fluids.WATER.getAttributes().getBlock(null,null,Fluids.WATER.getDefaultState());
+        assert state.getBlock() == Blocks.WATER && state2 == state;
```

Comment:
Please use Validate.isTrue instead of assert, no one enables assertions

Reply:
Right.